packages/manager/apps/telecom/src/app/telecom/telephony/billingAccount/orderAlias/order-alias-main.html

```diff
@@ -58,9 +58,19 @@ <h5 class="oui-heading_5">
                         data-ng-disabled="!offer.withoutTax"
                         data-uib-tooltip="{{ offer.tip | translate }}"
                         data-tooltip-placement="left"
+                        data-ng-if="offer.id !== 'special' || $ctrl.canOrderSpecial"
```

Comment:
Constant 'special' is used in multiple places. May consider moving it to constant file.